lib/living_document.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'active_support/core_ext/string/filters' # for `#squish`
 require 'memoist'
+# require 'stringio'
```

Comment:
Woops, this is not supposed to be commented out! Fixed in https://github.com/davidrunger/living_document/pull/66 .